containers/Home/Contact/Contact.tsx

```diff
@@ -1,13 +1,41 @@
 import { StyledContact } from './styles'
-import type { Props } from './interface'
+import type { Props, ContactProps } from './interface'
+import React from 'react'
+
+const handleType: React.FC<ContactProps> = ({ label, content, type }) => {
+    switch (type) {
+        case 'phone':
+            return (
+                <span>
+                    <a href={`tel:${content}`}>{content}</a>
+                </span>
+            )
+        case 'mail':
+            return (
+                <span>
+                    <a href={`mailto:${content}`} target="_blank" rel="noreferrer">
+                        {content}
+                    </a>
+                </span>
+            )
+        case 'skype':
+            return (
+                <span>
+                    <a href={`skype:${content}?chat`}>{content}</a>
+                </span>
+            )
+        default:
+            return <span>{content}</span>
+    }
+}
```

Comment:
This is React Component (as defined React.FC) so it should use the React naming convention

<span> tag has been duplicated a alot, can we refactor this? tbh, I think we could simplified the `switch case` solution to object mapping.